packages/flutter/lib/src/widgets/routes.dart

```diff
@@ -394,11 +396,8 @@ class _ModalScopeState extends State<_ModalScope> {
     });
   }
 
-  void _didChangeRouteOffStage() {
-    setState(() {
-      // We use the route's offstage bool in our build function, which means our
-      // state has changed.
-    });
+  void _routeSetState(VoidCallback fn) {
```

Comment:
it wouldn't help here, since the setState() we're calling isn't defined in this file.